package.json

```diff
@@ -45,5 +42,8 @@
     "remark": "14.0.3",
     "remark-mdx": "2.3.0"
   },
-  "packageManager": "yarn@3.6.4"
+  "packageManager": "yarn@3.6.4",
+  "engines": {
+    "node": ">=18.0"
```

Comment:
Same is in the `docusaurus` package: 

https://github.com/facebook/docusaurus/blob/2bb4fd0a6514cba424e5238e190a70626edf19d6/packages/docusaurus/package.json#L127